sky/resources.py

```diff
@@ -196,8 +194,15 @@ def instance_type(self):
         return self._instance_type
 
     @property
-    def cpu(self) -> Optional[str]:
-        return self._cpu
+    def cpus(self) -> Optional[str]:
+        """Returns the number of vcpus that each instance must have.
+
+        For example, cpus='4' means each instance must have exactly 4 vcpus,
+        and cpus='4+' means each instance must have at least 4 vcpus.
+        The cpus field is only used to select the instance type at launch time.
+        Thus, Resources in ResourceHandle will have cpus field set to None.
```

Comment:
```suggestion
        (Developer note: Thus, Resources in the backend's ResourceHandle will have cpus field set to None.)
```

Reply:
Good catch. Marked the last two sentences as a developer note. PTAL.